src/limits.cpp

```diff
@@ -450,7 +450,7 @@ void remove_patch(struct char_data *ch)
         act("You resist the feeble effects of $p.", FALSE, ch, patch, 0, TO_CHAR);
       break;
     case PATCH_TRAUMA:
-      if (success_test(GET_REAL_BOD(ch) - (GET_BIOOVER(ch) > 0 ? (GET_BIOOVER(ch) + 1) / 2 : 0), GET_OBJ_VAL(patch, 1)) > 0)
+      if (success_test(GET_REAL_BOD(ch) - (GET_BIOOVER(ch) > 0 ? (GET_BIOOVER(ch) + 1) / 2 : 0), GET_PATCH_RATING(patch)) > 0)
```

Comment:
These are my favorite types of changes, thanks for helping make the code more readable!